packages/aws-cdk-lib/aws-secretsmanager/README.md

```diff
@@ -25,7 +25,7 @@ const instance1 = new rds.DatabaseInstance(this, "PostgresInstance1", {
 // Templated secret with username and password fields
 const templatedSecret = new secretsmanager.Secret(this, 'TemplatedSecret', {
   generateSecretString: {
-    secretStringTemplate: JSON.stringify({ username: 'postgres' }),
+    secretStringTemplate: JSON.dumps({ username: 'postgres' }),
```

Comment:
This won't work because the example is in TypeScript. This example is translated to python using JSII. I'm not sure what we have to do to fix it though, I'll look into it.